### ISSUE TITLE
npx yoshi lint hangs 

### ISSUE THREAD

**Issue by mastertheblaster** [wix/yoshi#269]
# 🐛 Bug Report

### 😯 Current Behavior

Open `package.json`, create invalid eslint rule and run:

```bash
npx yoshi lint
```

The process remains hanging...

### 🤔 Expected Behavior

The process should exit with error code.

### 🔦 Context

https://github.com/wix-private/promote/tree/master/te-smart-actions-widget

### 🌍 Your Environment

```
➜  te-smart-actions-widget git:(master) npx yoshi info

  System:
    OS: macOS High Sierra 10.13.4
    CPU: x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
  Binaries:
    Node: 9.11.1 - ~/.nvm/versions/node/v9.11.1/bin/node
    Yarn: 1.3.2 - /usr/local/bin/yarn
    npm: 5.6.0 - ~/.nvm/versions/node/v9.11.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: 66.0.3359.139
    Firefox: 59.0.1
    Safari: 11.1
  npmPackages:
    yoshi: latest => 2.1.9

 Yoshi config
{
  "entry": {
    "app": "./client/index.js",
    "component": "./client/component.js",
    "manifest": "./client/manifest.js",
    "module": "./client/module.js"
  },
  "externals": {
    "react": "React",
    "react-dom": "ReactDOM",
    "react-module-container": "reactModuleContainer"
  }
}
```

**Comment by yavorsky** [comment]
Just tried to reproduce this, but unsuccessfully.
<img width="818" alt="screen shot 2018-05-10 at 3 06 39 pm" src="https://user-images.githubusercontent.com/1521229/39868856-127a2e5c-5464-11e8-9e30-29ff96e4b31e.png">
`npx yoshi lint` works w/o any errors. can you please:
`npm i -g qnm`;
and then in the root of the project try `qnm eslint`:
<img width="506" alt="screen shot 2018-05-10 at 3 06 47 pm" src="https://user-images.githubusercontent.com/1521229/39869101-eb72497e-5464-11e8-883a-be6efadb885a.png">

**Comment by ranyitz** [comment]
@yavorsky We've released a fix to this issue - https://github.com/wix/yoshi/issues/268 and that's why it's not reproducible that way.

I think that the problem here is the fact that when there is an error on in the lint task, the process hangs.

**Comment by mastertheblaster** [comment]
Yep Ran is 100% right.

The priority of this is very low, cause it's a configuration issue and very obvious.
Reported just in order `yoshi` would be `bugless`.

**Comment by ronami** [comment]
Should be solved by https://github.com/wix/haste/pull/190.

**Comment by ranyitz** [comment]
Fixed on [v2.7.0](https://github.com/wix/yoshi/blob/master/CHANGELOG.md#270-may-31-2018)
